### PR TITLE
docs(README): reorder releases latest first

### PR DESCRIPTION
**Pull request by Undistraction**
Makes more sense having latest release first, then descending order.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,47 +111,47 @@ We are building our npm distributions using Webpack to support legacy versions o
...
- [1.18.0](https://char0n.github.io/ramda-adjunct/1.18.0),
- [1.19.0](https://char0n.github.io/ramda-adjunct/1.19.0),
- [2.0.0](https://char0n.github.io/ramda-adjunct/2.0.0),
- [2.1.0](https://char0n.github.io/ramda-adjunct/2.1.0),
- [2.2.0](https://char0n.github.io/ramda-adjunct/2.2.0),
- [2.3.0](https://char0n.github.io/ramda-adjunct/2.3.0),
- [2.4.0](https://char0n.github.io/ramda-adjunct/2.4.0),
- [2.4.1](https://char0n.github.io/ramda-adjunct/2.4.1),
- [2.5.0](https://char0n.github.io/ramda-adjunct/2.5.0),
  [LATEST](https://char0n.github.io/ramda-adjunct)
```
Comment (guillaumearm) [inline README.md]: add comma

File: README.md [hunk 2]
```diff
@@ -111,47 +111,47 @@ We are building our npm distributions using Webpack to support legacy versions o
...
+ [1.1.0](https://char0n.github.io/ramda-adjunct/1.1.0),
+ [1.0.0](https://char0n.github.io/ramda-adjunct/1.0.0),
+ [0.7.0](https://char0n.github.io/ramda-adjunct/0.7.0),
+ [0.6.0](https://char0n.github.io/ramda-adjunct/0.6.0),
+ [0.5.1](https://char0n.github.io/ramda-adjunct/0.5.1),
+ [0.4.0](https://char0n.github.io/ramda-adjunct/0.4.0),
+ [0.3.0](https://char0n.github.io/ramda-adjunct/0.3.0),
+ [0.2.0](https://char0n.github.io/ramda-adjunct/0.2.0),
+ [0.1.0](https://char0n.github.io/ramda-adjunct/0.1.0),
+ [0.0.1](https://char0n.github.io/ramda-adjunct/0.0.1),
```
Comment (guillaumearm) [inline README.md]: remove comma

Comment (Undistraction) [reply README.md]: Thanks. If there is one thing that constantly surprises me about this line of work, it's how easy it is to miss the glaringly obvious.

Comment (guillaumearm) [reply README.md]: yeah, in programming, mistakes are ~always~ often dumb. That's why code review is a very good thing to do ;-)

**Pull request merged**